ext/telemetry/lib.rs

```diff
@@ -851,6 +911,9 @@ macro_rules! attr_raw {
     } else if let Ok(bigint) = $value.try_cast::<v8::BigInt>() {
       let (i64_value, _lossless) = bigint.i64_value();
       Some(Value::I64(i64_value))
+    } else if let Ok(_array) = $value.try_cast::<v8::Array>() {
+      // TODO: implement array attributes
```

Comment:
Open an issue about it before landing?

Reply:
https://github.com/denoland/deno/issues/27552